### ISSUE TITLE
[a11y] Buttons do not have an accessible name

### ISSUE THREAD

**Issue by rjbeers** [codeconnector/codeconnector.github.io#27]
>When a button doesn't have an accessible name, screen readers and other assistive technologies announce it as button, which provides no information to users about what the button does.

See this [lighthouse report](https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fcodeconnector.io%2F#accessibility) for failures on the landing page.

[web.dev](https://web.dev/button-name/) has more details on this failure and how to solve it.

**Comment by rjbeers** [comment]
Fixed by #30
